src/Session/SessionAwareTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Pantheon\Terminus\Session;
+
+
+/**
+ * Provides the basic properties needed to fulfill the SessionAwareInterface.
+ * 
+ * Class SessionAwareTrait
+ * @package Pantheon\Terminus\Session
+ */
+trait SessionAwareTrait {
+
+  /**
+   * @var Session
+   */
+  protected $session;
+
+  /**
+   * @inheritdoc
+   */
+  public function setSession(Session $session)
+  {
+    $this->session = $session;
+
+    return $this;
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public function session()
+  {
+    if (!isset($this->session)) {
+      $this->setSession(new Session());
```

Comment:
This is kind of a violation of the DI principal right here. It's ok for now until we set up our inflector to do the injection properly IMHO.